parse/DoubleComplexValueRefParser.cpp

```diff
@@ -26,6 +26,13 @@ namespace parse {
                   )     [ _val = new_<ValueRef::ComplexVariable<double> >(_a, _b, _c, _f, _d, _e) ]
                 ;
 
+            empire_meter_value
```

Comment:
This change is the small brother of SetEmpireMeter. 

EmpireDetection etc.. should probably also be done.
But not in this PR.